src/gc/gcenv.ee.standalone.inl

```diff
@@ -11,166 +11,196 @@
 // will be fowarded to this interface instance.
 extern IGCToCLR* g_theGCToCLR;
 
+// A note about this:
+// In general, we don't want to pretend to be smarter than the compiler
+// and force it to inline things. However, inlining is here is required
+// for correctness as it stands today (though it will not always be required).
```

Comment:
It should not be a problem as long as you build the GC so correctly and only export the few symbols that should be exported from it.